.circleci/config.yml

```diff
@@ -488,19 +479,14 @@ jobs:
           no_output_timeout: 30m
 
   spm-revenuecat-ui-ios-15:
-    <<: *base-job
-    parameters:
-      xcode_version:
-        type: string
-        default: '14.3.0'
-      install_swiftlint:
-        type: boolean
-        default: false
+    executor:
+      name: macos-executor
+      xcode_version: '14.3.0'
```

Comment:
again, this just looks nicer, and you don't have to redefine the parameter every time including the type